app/src/main/res/drawable/plus_button.xml

```diff
@@ -0,0 +1,7 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Zaujimave riesenie ;) Ja som vzdy hladal pngcka na nete. Nice!

Reply:
https://materialdesignicons.com/
Da sa to odtial zkopirovat ako drawable aj ako png